app/messages.py

```diff
@@ -123,6 +123,9 @@
                                              " config value has to be within"
                                              " these values: prod,"
                                              "dev, test."}
+CURRENT_PASSWORD_MATCHES_NEW_PASSWORD = {"message": "The password has not been "
+                                         "changed. You can not use same "
+                                         "password as current one."}
```

Comment:
Please a space at the beginning of lines 127 and 128 and not the end of the lines 126/127 respectively inside the string.